app/models/contestant.rb

```diff
@@ -1,3 +1,13 @@
 class Contestant < ApplicationRecord
   validates_presence_of :name, :age, :hometown, :years_of_experience
+  has_many :contestant_projects
+  has_many :projects, through: :contestant_projects
+
+  def render_projects
+    names = []
+    self.projects.each do |project|
+      names << project.name
+    end
+    names.join(", ")
```

Comment:
Hmm.... So, this is going through all the `projects` that belong to this one Contestant and saying, "just gimme the names of each of your projects". This is very much the ruby way of doing it; in ActiveRecord we should use `.pluck`. 

I think you messaged us about using `.join(", ")` which I approved of at the time, but I thought you'd be using it in the view _after_ doing .pluck, sorry if I misled you here! 

If we rely on AR for this method, it turns into something like: 

```
def render_projects
    projects.pluck(:name)
end
```

Reply:
Maybe a red flag for you to keep in mind would be, if you ever feel like doing a shovel `<<` operation in a model method, you can almost guaranteed find an AR alternative!